src/Main.elm

```diff
@@ -923,9 +923,6 @@ updateFormField msg fieldIndex string formFields formField =
                                         [] ->
                                             AttributeInvalid string
 
-                                        [ _ ] ->
-                                            AttributeInvalid string
-
                                         list ->
                                             AttributeGiven (List.map choiceFromString list)
```

Comment:
bad call in the first place https://github.com/choonkeat/tiny-form-fields/pull/10